From 17ecdacf77dbf5e263de34fef5d00a8d0793f4df Mon Sep 17 00:00:00 2001
From: Jianqun Xu <jay.xu@rock-chips.com>
Date: Wed, 27 Jul 2016 17:23:08 +0800
Subject: [PATCH] ARM64: dts: rk3399-evb3: ajust gpu opp table

Some evb3 need 1v for 800M.
We remove 700M currently since havn't a good opinion of it.

Change-Id: I7083857f5bb8a09efe0369a6bdd49e28c67da8b7
Signed-off-by: Jianqun Xu <jay.xu@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-evb-rev3.dtsi | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3.dtsi
index 6c2f27acf40f..491edc0309aa 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3.dtsi
@@ -220,13 +220,9 @@
 		opp-hz = /bits/ 64 <600000000>;
 		opp-microvolt = <900000>;
 	};
-	opp@700000000 {
-		opp-hz = /bits/ 64 <700000000>;
-		opp-microvolt = <925000>;
-	};
 	opp@800000000 {
 		opp-hz = /bits/ 64 <800000000>;
-		opp-microvolt = <950000>;
+		opp-microvolt = <1000000>;
 	};
 };
 
-- 
2.35.3

